lib/engine-native/src/engine.rs

```diff
@@ -27,6 +29,10 @@ impl NativeEngine {
     /// Create a new `NativeEngine` with the given config
     #[cfg(feature = "compiler")]
     pub fn new(compiler: Box<dyn Compiler + Send>, target: Target, features: Features) -> Self {
+        if let Err(_) = which("gcc").or_else(|_| which("clang-10")) {
```

Comment:
We should try (by this order):
* `clang-10` (cross compile)
* `clang` (cross compiled)
* `gcc`